fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/EverythingOperationTenant1Test.java

```diff
@@ -0,0 +1,290 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.server.test.operation;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.AssertJUnit.assertFalse;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.SerializationUtils;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.core.FHIRMediaType;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Bundle.Entry;
+import com.ibm.fhir.model.resource.CapabilityStatement;
+import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.test.TestUtil;
+import com.ibm.fhir.model.type.code.BundleType;
+import com.ibm.fhir.path.FHIRPathNode;
+import com.ibm.fhir.path.evaluator.FHIRPathEvaluator;
+import com.ibm.fhir.path.evaluator.FHIRPathEvaluator.EvaluationContext;
+import com.ibm.fhir.server.test.FHIRServerTestBase;
+
+/**
+ * Tests the Everything Operation
+ */
+public class EverythingOperationTenant1Test extends FHIRServerTestBase {
+
+    private static final String CLASSNAME = EverythingOperationTest.class.getName();
+    private static final Logger LOG = Logger.getLogger(CLASSNAME);
+
+    public static final String EXPRESSION_OPERATION = "rest.resource.operation.name";
+
+    private static Boolean SKIP = null;
+
+    private static final boolean DEBUG = false;
+
+    private Map<String, List<String>> createdResources;
+    private String patientId;
+
+    public EverythingOperationTenant1Test() {
+        createdResources = new HashMap<>();
+    }
+
+    @BeforeClass
+    public void checkOperationExistsOnServer() throws Exception {
+        if (SKIP == null) {
+            CapabilityStatement conf = retrieveConformanceStatement();
+            FHIRPathEvaluator evaluator = FHIRPathEvaluator.evaluator();
+            EvaluationContext evaluationContext = new EvaluationContext(conf);
+            // All the possible required operations
+            Collection<FHIRPathNode> tmpResults = evaluator.evaluate(evaluationContext, EXPRESSION_OPERATION);
```

Comment:
if we want to be more precise and look for just Patient/$everything (instead of any resource operation named everything), I think we could use `rest.resource.where(type='Patient').operation.name` instead of `rest.resource.operation.name`

Reply:
or `rest.resource.where(type='Patient').operation.name.value` if we want a collection of string values back (would just let us remove the `getValue()` from `x.getValue().asStringValue().string()` on 72 I think

Reply:
I did the top one.  Done.